apps/client/src/app/pages/checklist/checklist/checklist.component.ts

```diff
@@ -79,25 +90,17 @@ export class ChecklistComponent {
 
   public tasks$: Observable<LostarkTask[]> = combineLatest([
     this.rawRoster$,
-    this.tasksService.tasks$
+    this.tasksService.tasks$,
+    this.roster$,
```

Comment:
Don't add a new Observable to the combineLatest operator if it's going to emit at the same time as another one, in this case, characters is really just here to map from rawRoster and add className, which shouldn't be done here and will create an ambiguous situation in database where some characters have className and some do not, depending on when they were last saved.